evaluate.py

```diff
@@ -106,6 +110,14 @@ def run_eval_loop(
             y = batch["mask"].to(device)
         elif "label" in batch:
             y = batch["label"].to(device)
+        elif "boxes" in batch:
+            y = [
+                {
+                    "boxes": batch["boxes"][i].to(device),
+                    "labels": batch["labels"][i].to(device),
+                }
+                for i in range(len(batch["image"]))
```

Comment:
I wonder if `torchgeo.datasets.unbind_samples` could help here

Reply:
Think it could. I'll take a look.

Reply:
It can work but we will still need to iterate over them and move them to specified device.